src/IO/ResampleLabelImage/resampleLabelImage.js

```diff
@@ -0,0 +1,58 @@
+import { runPipeline, InterfaceTypes, imageSharedBufferOrCopy } from 'itk-wasm'
+
+import itkConfig from '../itkConfig.js'
+
+async function runWasm(pipeline, args, image) {
+  const taskArgs = ['0', '0', ...args, '--memory-io']
+  const inputs = [
+    {
+      type: InterfaceTypes.Image,
+      data: imageSharedBufferOrCopy(image),
+    },
+  ]
+
+  const desiredOutputs = [{ type: InterfaceTypes.Image }]
+
+  const { pipelinesUrl, pipelineWorkerUrl } = itkConfig
+
+  // remove blob:http://... added to __webpack_public_path__ in webworker
+  const pipelinesUrlNoBlob = pipelinesUrl.startsWith('blob:')
+    ? pipelinesUrl.substring(5)
+    : pipelinesUrl
+
+  const pipelinesWorkerUrlNoBlob = pipelineWorkerUrl.startsWith('blob:')
+    ? pipelineWorkerUrl.substring(5)
+    : pipelineWorkerUrl
+
+  const worker = new Worker(
+    new URL(pipelinesWorkerUrlNoBlob, self.location.origin) // URL is invalid for tests unless base provided to URL
+  )
+
+  const { outputs } = await runPipeline(
+    worker,
```

Comment:
Using `null` here will generate a worker using the configuration from `itkConfig`.

Reply:
For the worker, the `__webpack_public_path__` starts with `blob:` for something like `blob:http://...`.  So now overriding `itkConfig.pipelinesUrl` and `itkConfig.pipelineWorkerUrl` after stripping `blob:` off.  

For `npm run test` to work, now composing the relative `/base/dist/...` in the itkConfig with the `self.location.origin` avoids a XMLRequest error about an incomplete URL.

Now can use `null` in runPipeline and leave `pipelinesUrl` parameter off.  This makes WorkerPool happy too so running the split logic now too.